cpp/CMakeLists.txt

```diff
@@ -340,15 +355,28 @@ message(STATUS "CUDF_TEST_LIST set to: ${CUDF_TEST_LIST}")
 
 if(BUILD_BENCHMARKS)
 
-  include(ConfigureGoogleBenchmark)
-
-  if(GBENCH_FOUND)
-    message(STATUS "Google C++ Benchmarking Framework (Google Benchmark) found in ${GBENCH_ROOT}")
+    if(GBENCH_ROOT)
+        message(STATUS "Google C++ Benchmarking Framework (Google Benchmark) found in ${GBENCH_ROOT}")
+        add_subdirectory(${CMAKE_SOURCE_DIR}/benchmarks)
+    else()
+        message(AUTHOR_WARNING "Google C++ Benchmarking Framework (Google Benchmark) not found, it will be built from source.")
+        include(ConfigureGoogleBenchmark)
+        add_subdirectory(${CMAKE_SOURCE_DIR}/benchmarks)
+        set(GBENCH_ROOT "${CMAKE_BINARY_DIR}/googlebenchmark")
+    endif(GBENCH_ROOT)
```

Comment:
Should be able to do the same here with `find_package(benchmark)`

Reply:
This step is required if it's not found to build it. As I removed that include from the top of the bench setup and put it here. I had issues with `findpackage` working after checkout and building even when linked. With both gtest and benchmarks. `findpackage` would claim it couldn't find it even with hints. Are you saying the whole thing could be replaced or just the variable setting and subdirectory?

Reply:
Did you install the `benchmark` conda package? Otherwise your existing environment files don't have google benchmark being installed.

Reply:
Added. But I was also solving for situations when benchmark has not been installed by conda.